types/chrome/index.d.ts

```diff
@@ -6245,7 +6245,7 @@ declare namespace chrome {
         }
 
         export interface MenuItemParameters {
-            items: Object[];
+            items: object[];
```

Comment:
This should be [`MenuItem[]`](https://developer.chrome.com/docs/extensions/reference/api/input/ime#type-MenuParameters)